solr/bin/solr

```diff
@@ -112,6 +112,13 @@ elif [ -r "$SOLR_INCLUDE" ]; then
   . "$SOLR_INCLUDE"
 fi
 
+# Export all SOLR variables so they are visible to Solr sub-process
+for var in $(compgen -e); do
```

Comment:
`compgen` is new to me...    I had to look a bit to learn, really interesting how short it works.

Reply:
Me too, ChatGPT surprised me there 😉